src/audio/tdfb/tdfb.c

```diff
@@ -443,6 +655,15 @@ static int tdfb_copy(struct comp_dev *dev)
 			     n * cl.sink_frame_bytes);
 	}
 
+	/* Update every 1s the the direction angle */
+	//cd->count++;
```

Comment:
TODO?

Reply:
Ah, this was an experiment to simulate the direction of arrival tracking. It needs a kernel patch to pass the request commad to firmware when DSP is awake. Instead of 1s period here, it would be probably an always running band-pass level meter (for typical speech frequencies) that triggers the angle calculation. Later it could be proper VAD triggered.